app/services/PensionSessionService.scala

```diff
@@ -168,10 +168,34 @@ class PensionSessionService @Inject()(pensionUserDataRepository: PensionsUserDat
         //TODO: set the question below based on the list from backend
         uKPensionIncomesQuestion = Some(getUkPensionIncome(prior).nonEmpty),
         uKPensionIncomes = getUkPensionIncome(prior)
+      ),
+
+      unauthorisedPayments = UnauthorisedPaymentsViewModel(
+        unauthorisedPaymentsQuestion = getUnauthorisedPaymentsQuestion(prior),
```

Comment:
I like how you're using a function here; much more concise than how we do it below.

Perhaps which should do the same with those, sometime?